src/geotiff-validator/README.md

```diff
@@ -0,0 +1,63 @@
+# GeoTIFF Validator
+
+## Worker Configuration
+
+Initially, the worker reads the default configuration file in `./config/config.default.yml`. This can be adapted to your needs - see the following list:
+
+**Validation steps:**
+
+|key    | description|
+|-----  |------------|
+|projection |Allowed projection|
+|fileSize |Allowed file size|
+|dataType |Allowed data type|
+|extent |Allowed extent|
```

Comment:
the extend is still dependent on the projcetion?
maybe we say the extent is always EPSG:4326 or we add the extent as property of the projection :thinking: